libyul/AssemblyStack.cpp

```diff
@@ -226,7 +226,7 @@ MachineAssemblyObject AssemblyStack::assemble(Machine _machine) const
 	return MachineAssemblyObject();
 }
 
-pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assembleAndGuessRuntime() const
+std::pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assembleWithDeployed(optional<string_view> _runtimeName) const
```

Comment:
The term "runtime" keeps on creeping into our codebase, but it is not correct. Please at least do not try to re-introduce it. It should be named "deployed".